src/LibKubernetesGenerator/templates/ModelExtensions.cs.template

```diff
@@ -1,17 +1,17 @@
 // <auto-generated>
 // Code generated by https://github.com/kubernetes-client/csharp/tree/master/gen/KubernetesGenerator
-// Changes may cause incorrect behavior and will be lost if the code is
-// regenerated.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
 // </auto-generated>
 namespace k8s.Models
 {
 {{#.}}
-    [KubernetesEntity(Group="{{GetGroup . }}", Kind="{{GetKind . }}", ApiVersion="{{GetApiVersion . }}", PluralName={{GetPlural .}})]
+    [KubernetesEntity(Group=KubeGroup, Kind=KubeKind, ApiVersion=KubeApiVersion, PluralName=KubePluralName)]
```

Comment:
With this change, less hardcoded strings are embedded, rather a const string is reused.